3rdparty/7z/CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ if(WIN32)
 	target_include_directories(3rdparty_7z INTERFACE
 		$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
 		$<INSTALL_INTERFACE:/src>)
+
+	target_include_directories(3rdparty_7z INTERFACE 7z)
+
+	set_property(TARGET 3rdparty_7z PROPERTY FOLDER "3rdparty/")
+
 else()
 	add_library(3rdparty_7z INTERFACE)
-endif()
```

Comment:
Missing newline

Reply:
fixed, but it still doesnt show the newline on github pages

Reply:
If the file ends in a newline then it's not shown. It's just a linux way of marking the end of the file; if a line ends abruptly in middle then that's a sign that it might have not been fully transferred

Reply:
fair enough